lib/index.js

```diff
@@ -282,7 +288,7 @@ api.createIndexes = brCallbackify(async options => {
  * @return the new GridFSBucket instance
  */
 api.createGridFSBucket = options => {
-  const opts = Object.assign({}, {writeConcern: api.writeOptions}, options);
+  const opts = {...options, ...api.writeOptions};
```

Comment:
Also, we should not use `api.writeOptions` since it doesn't get updated when the config changes (and we're deprecating it ... so it shouldn't be used anywhere).

```suggestion
  const opts = {...bedrock.config.mongodb.writeOptions, ...options};
```